app/graphql/mutations/create_response_mutation.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Mutations
+  class CreateResponseMutation < BaseMutation
+    description "Create a new survey's response"
+
+    argument :response, Types::Inputs::ResponseInput, required: true
+
+    field :success, Boolean, null: false
+
+    def resolve(response:)
+      response_form = ResponseForm.new(response)
+
+      raise(GraphQL::ExecutionError, response_form.errors.full_messages.to_sentence) unless response_form.save
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/096a0b3446faa9f2f3f01faae7a30306d5fe7136/app/graphql/mutations/create_response_mutation.rb#L14"></span>Refers to ‘response_form’ more than self (maybe move it to another class?)</td>
    </tr>
  </tbody>
</table>